shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -773,8 +775,19 @@ private void onPreEngineRestart() {
       listener.onPreEngineRestart();
     }
   }
-  // ----- End Engine Lifecycle Support ----
 
+  @SuppressWarnings("unused")
+  @UiThread
+  private void onDisplayPlatformView(int viewId, int x, int y, int width, int height) {
```

Comment:
move this method below `// ----- End Engine Lifecycle Support ----` since `onDisplayPlatformView` isn't related to the engine lifecycle.

Reply:
updated